cmd/controller/main.go

```diff
@@ -32,9 +32,10 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 
 	metal3iov1alpha1 "github.com/metal3-io/baremetal-operator/apis/metal3.io/v1alpha1"
-	metal3iocontroller "github.com/openshift/image-customization-controller/controllers/metal3.io"
+	metal3iocontroller "github.com/metal3-io/baremetal-operator/controllers/metal3.io"
```

Comment:
Should this be openshift/baremetal-operator/controllers/metal3.io ?

Reply:
No, the import name has to match [the `go.mod` file](https://github.com/openshift/baremetal-operator/blob/master/go.mod#L1) otherwise go will complain.
It's the [replace directive in the local `go.mod` file](https://github.com/openshift/image-customization-controller/blob/main/go.mod#L26) that ensures we use the downstream fork.

Reply:
(I know because I tried it this morning and go complained :laughing:)

Reply:
Ah! Good to know